datadog/resource_datadog_synthetics_test_test.go

```diff
@@ -457,6 +457,9 @@ resource "datadog_synthetics_test" "ssl" {
 	options = {
 		tick_every = 60
 		accept_self_signed = true
+		monitor_options {
+			renotify_interval = 20
+		}
```

Comment:
When trying to run tests with this modification, I'm getting the following error:
```
=== RUN   TestAccDatadogSyntheticsSSLTest_Basic
--- FAIL: TestAccDatadogSyntheticsSSLTest_Basic (0.02s)
    testing.go:640: Step 0 error: config is invalid: Incorrect attribute value type: Inappropriate value for attribute "options": element "monitor_options": string required.
```

I believe that this might be a problem with too much nesting which TF can't handle, but I'm not sure. Maybe you could try putting the `renotify_interval` attribute directly inside `options` map and call it `monitor_renotify_interval`?